### PR TITLE
Address #317: Add x86_64 for simulator build

### PR DESCRIPTION
**Pull request by jrabek**
Also cleans up an umbrella header

### CODE REVIEW

**Review by ricardopsantos — APPROVED** [review]

File: libPhoneNumber.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -61,7 +61,6 @@
 		940725B624BF90290011AE05 /* NBShortNumberMetadataHelper.m in Sources */ = {isa = PBXBuildFile; fileRef = 940725B424BF90290011AE05 /* NBShortNumberMetadataHelper.m */; };
 		9439832824CB545000CF8CB4 /* README.md in Resources */ = {isa = PBXBuildFile; fileRef = 9439832724CB545000CF8CB4 /* README.md */; };
 		9439832D24CB569100CF8CB4 /* README.md in Resources */ = {isa = PBXBuildFile; fileRef = 9439832C24CB569100CF8CB4 /* README.md */; };
-		944B9B022497ED79007E17B5 /* NBGeocoderMetadataHelper.h in Headers */ = {isa = PBXBuildFile; fileRef = 944B9B002497ED79007E17B5 /* NBGeocoderMetadataHelper.h */; settings = {ATTRIBUTES = (Public, ); }; };
```
Comment (paween) [inline libPhoneNumber.xcodeproj/project.pbxproj:64]: Curious why is this removed?

Comment (jrabek) [reply libPhoneNumber.xcodeproj/project.pbxproj:64]: It was a public header in the wrong target which made any project that imported libPhoneNumber that the umbrella header was missing this header.  Not sure why Xcode didn't complain before?

Comment (olejnjak) [reply libPhoneNumber.xcodeproj/project.pbxproj:64]: Effectively I think, that this change made other schemes [uncompilable](https://travis-ci.org/github/iziz/libPhoneNumber-iOS/builds/738085136)
